CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```

Comment:
This was generated from a template GitHub provided. I think this is a good start/overview, so lmk if there's things you'd recommend changing!